var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -68,7 +69,7 @@ class Qmcpack(CMakePackage):
     variant('gui', default=False,
             description='Install with Matplotlib (long installation time)')
     variant('qe', default=True,
-            description='Install with patched Quantum Espresso 5.3.0')
+            description='Install with patched Quantum Espresso 6.3.0')
```

Comment:
Is this the only version that works? It would be nice if it supported a whole range of versions.

Reply:
Now that QE has a supported HDF5 interface, supporting multiple versions of QE will be more practical in future. We will be supporting at least the last two significant versions of QE.

Reply:
If I understand you correctly, then the supported version of quantum espresso depends on the version of qmcpack -> this should be reflected in the depends_on's below

Reply:
You are misunderstanding. QE is neither a build time nor a run time dependency. It is part of a larger QMCPACK workflow. Output from QE can be used as input to QMPACK. Many science codes have similarly workflows.

For user convenience, there is a QE variant and we apply our patch to it. The dependency is there when the variant is on.